draft-packages/title-block-zen/KaizenDraft/TitleBlockZen/TitleBlockZen.spec.tsx

```diff
@@ -642,6 +643,32 @@ describe("<TitleBlockZen />", () => {
     })
   })
 
+  describe("survey status", () => {
+    it("it doesn't render tag when field is omitted", async () => {
+      render(<TitleBlockZen title="Test Title">Example</TitleBlockZen>)
+
+      await waitFor(() => {
+        expect(
+          screen.queryByTestId("survey-status-tag")
+        ).not.toBeInTheDocument()
+      })
+    })
+    it("it renders tag with correct text and variant when draft status", async () => {
```

Comment:
@juliohr should we also test for the `scheduled` and `closed` statuses, given this is the functionality being added by the PR?